ledger-core/v2/reference/scope.go

```diff
@@ -27,47 +31,79 @@ const ( // super-scopes
 )
 
 const ( // super-scopes
-	LifelineSuperScope    Scope = 0x04 * baseScopeLifeline
-	LocalDomainSuperScope Scope = 0x04 * baseScopeLocalDomain
-	GlobalSuperScope      Scope = 0x04 * baseScopeGlobal
+	superScopeLifeline    Scope = 0x04 * baseScopeLifeline
```

Comment:
I would give name to 0x04

Reply:
done
593407f8c87af0ebb5d4301e5e2983510f2aad7d